packages/cli/lib/preview.ts

```diff
@@ -28,25 +30,47 @@ export default async (config: CliOptions): Promise<void> => {
       return parser(content)
     }
 
-    const app = await carlo.launch()
-    app.on('exit', () => process.exit())
-    app.serveFolder(__dirname + '/templates/preview')
-
-    class Events extends EventEmitter {}
-    const event = new Events()
-    await app.exposeFunction('event', () => event)
-    await app.exposeFunction('generate', async () => {
-      return await generate()
+    // get the Chromium path
+    const executablePath = Launcher.getFirstInstallation()
+    const browser = await puppeteer.launch({
+      executablePath,
+      headless: false,
+      defaultViewport: null,
+      ignoreDefaultArgs: ['--disable-extensions'],
+      args: [
+        '--start-maximized',
+        `--app=data:text/html,<title>${encodeURIComponent('Vuese CLI')}</title>`
+      ],
     })
+    await browser.target().createCDPSession()
+
+    // get all of browser tab
+    const pages = await browser.pages()
+    // use default or create a browser tab
+    const page = pages && pages.length > 0 ? pages[0] : await browser.newPage()
```

Comment:
since typescript 3.7, you can write like this, 
```ts
const page = pages?.length ? pages[0] : await browser.newPage();
```

Reply:
got it